screens/HomeScreen.tsx

```diff
@@ -205,12 +212,15 @@ const HomeScreen = ({
       const name = token ? token.name : "--------";
       const decimals = token ? token.decimals : null;
       const amountFormatted = formatAmount(amount, decimals);
-      return {
+
+      const obj = {
         symbol,
         name,
         amount: amountFormatted,
         tokenId
       };
+
+      return obj;
```

Comment:
Adding an extra object assignment here, then passing it immediately.  
Either remove this extra assignment, or give it a meaningful name; `obj` means nothing.